doc/nrf/ug_thread_supported_features.rst

```diff
@@ -82,3 +82,27 @@ Developers can make use of the OpenThread API to obtain information about the li
 Enhanced-ACK probing is the most power efficient method to retrieve link metrics since very little message overhead is required.
 |NCS| provides full Link Metrics support even for the RCP architecture, which is the most technically challenging one since the radio driver must handle the injection of Information Elements on time to match the acknowledgment timing requirements.
 The decision on how to interpret the link metrics information to adjust the transmission power is left to the application itself.
+
+Thread Domain Unicast Addressing
+================================
+
+Thread 1.2 Specification introduces the concept of Thread Domains.
+
+A Thread Domain is a set of Thread Devices that receive and apply a common Thread Domain operational configuration.
+The Thread Domain operational configuration enables Thread Devices to join and participate in larger interconnected systems extending beyond the limits of a single Thread Network.
```

Comment:
Thread Network or Thread network?
Normally I would say Thread network.
Unless there is a reason for doing otherwise.
I see we are using both ways of writing in this section.
We should align.

Reply:
Let's go with _Thread network_.